auth/tests/unit/cognito-throttles-cloudwatch-alarms.test.ts

```diff
@@ -247,18 +268,18 @@ describe.each(testCases)(
       expect(cloudWatchAlarmUnderTest.Properties.MetricName).toEqual(
         metricName
       );
-      expect(cloudWatchAlarmUnderTest.Properties.Namespace).toEqual(
-        "AWS/Cognito"
-      );
-      expect(cloudWatchAlarmUnderTest.Properties.Statistic).toEqual(statistic);
-      expect(cloudWatchAlarmUnderTest.Properties.Period).toEqual(period);
-      expect(cloudWatchAlarmUnderTest.Properties.EvaluationPeriods).toEqual(
-        evaluationPeriods
-      );
-      expect(cloudWatchAlarmUnderTest.Properties.DatapointsToAlarm).toEqual(
-        datapointsToAlarm
-      );
-      expect(cloudWatchAlarmUnderTest.Properties.Threshold).toEqual(threshold);
+
+      if (metricName.includes("WAF")) {
+        expect(cloudWatchAlarmUnderTest.Properties.Namespace).toEqual("AWS/WAFV2");
+      }
+      else {
+        expect(cloudWatchAlarmUnderTest.Properties.Namespace).toEqual("AWS/Cognito");
+      }
```

Comment:
Ideally we should avoid conditionals in tests ([eslint error](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-conditional-in-test.md)) - could we not just assert the conditions instead?

```ts
expect(metricName.includes("WAF")).toBe(true)
```

Reply:
This can be abstracted out into its own property in the AlarmTestCase type if needed, and passed as part of the array of test cases